File: CarsMatterApp/app/src/main/java/com/andrey/carsmatter/models/RefillNotesReport.java
package com.andrey.carsmatter.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RefillNotesReport {
    @SerializedName("totalCost")
    @Expose
    public float TotalCost;

    @SerializedName("costPerDay")
    @Expose
    public float CostPerDay;

    @SerializedName("costPerKm")
    @Expose
    public float CostPerKm;

    @SerializedName("totalVolume")
    @Expose
    public float TotalVolume;

    @SerializedName("averageVolume")
    @Expose
    public float AverageVolume;
}

File: CarsMatter.Infrastructure/Models/MsSQL/RefillNotesReport.cs
namespace CarsMatter.Infrastructure.Models.MsSQL
{
    public class RefillNotesReport
    {
        public float TotalCost { get; set; }

        public float CostPerDay { get; set; }

        public float CostPerKm { get; set; }

        public float TotalVolume { get; set; }

        public float AverageVolume { get; set; }
    }
}
